tiledbcontents/paths.py

```diff
@@ -68,7 +68,7 @@ def is_remote(path: str) -> bool:
     >>> is_remote("my/home/directory")
     False
     """
-    return split(path)[0] == "cloud"
+    return split(strip(path))[0] == "cloud"
```

Comment:
I wonder if it makes sense to a helper that does both split and strip since we use it in 3 different places.

Reply:
I had considered it, but there are places where we want to split without stripping. Also whatever it is would be like `split_strip` which saves 1? character maybe.

thanks for the quick review!